app/src/main/java/org/schabi/newpipe/player/helper/MediaSessionManager.java

```diff
@@ -40,13 +49,52 @@ public KeyEvent handleMediaButtonIntent(final Intent intent) {
         return MediaButtonReceiver.handleIntent(mediaSession, intent);
     }
 
+    @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
+    public void setLockScreenArt(
+            NotificationCompat.Builder builder,
+            @Nullable Bitmap thumbnailBitmap
+    ) {
```

Comment:
I would prefer if all the function declaration were on the same line, since there are only two arguments, and you don't come across many multiline declarations in the NewPipe codebase. But this is not really important ;-)